File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int sCtn = 0;
        int yCtn = 0;
        s = s.toUpperCase();
        
        for(int i = 0; i < s.length(); i++)
        {
            if(s.charAt(i) == 'P')
                sCtn++;
            else if(s.charAt(i) == 'Y')
                yCtn++;
        }

        if(sCtn == yCtn)
            return true;
        else
            return false;
    }
}

File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    s=s.toUpperCase();
    
    let pCtn = 0;
    let yCtn = 0;
    for(let i = 0; i < s.length; i++){
        console.log(s[i]);
        if(s[i] == 'P')
            pCtn++;
        else if(s[i] == 'Y')
            yCtn++;
    }
    if(pCtn === yCtn)
        return true;
    else
        return false;
}